storybookjs/storybook
# Build: Migrate `@storybook/web-components-vite` to strict TS

Part of #22176

## What I did

lint(tsconfig.json): change `compilerOptions.strict` from `false` => `true`

## How to test

0. ensure I'm using node 16
1. `yarn task --task compile --no-link --start-from=install`
2. `cd` into `code/frameworks/web-components-vite/`
3. Run `yarn check`

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
